bundle-generator/deployment/src/main/java/io/quarkiverse/operatorsdk/bundle/deployment/BundleProcessor.java

```diff
@@ -495,6 +488,19 @@ private CSVMetadataHolder createMetadataHolder(AnnotationInstance csvMetadata, C
                 origin);
     }
 
+    private Map<String, String> extractNameValueMap(AnnotationValue labelsField) {
```

Comment:
Nitpicking: can you rename the parameter and variables more generically, please?

Reply:
omg, thx